src/app/calculator/calculator.component.ts

```diff
@@ -14,65 +15,160 @@ import { MOCK } from '../shared/mockdata';
 export class CalculatorComponent {
   constructor() {
     this.motocycles = MOCK.models;
-    this.canvasLeg = new p5(this.sketch, 'canvas-human');
+    this.canvasLeg = new p5(this.sketchLeg, 'canvas-human');
     this.canvasLeg.setup = () => {
-      this.canvasLeg.createCanvas(1000, 560);
+      this.canvasLeg.createCanvas(1000, 660).position(0, 0);
       this.canvasLeg.noLoop();
     };
+    this.canvasTors = new p5(this.sketchTorsAndArm, 'canvas-human');
+    this.canvasTors.setup = () => {
+      this.canvasTors.createCanvas(1000, 660).position(0, 0);
+      this.canvasTors.noLoop();
+    };
   }
 
+  canvasLeg;
```

Comment:
for what this variable use this?

Reply:
to create canvas and draw leg

Reply:
but why you need this. It not looks like declaration

Reply:
it’s just a variable that will create an object p5 and then will be called to create a canvas, clear it, draw and transfer data.
What exactly do I need to change?

Reply:
I mean why do you need this in that place. Is it declaration?

Reply:
yes

Reply:
I got what you said, I changed to "canvas: p5;" :)